cmd/root.go

```diff
@@ -51,6 +54,11 @@ func init() {
 	cobra.OnInitialize(initConfig)
 
 	rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/config/stellar/config.yaml)")
+
+	// Add sub commands
+	rootCmd.AddCommand(auth.AuthCmd)
```

Comment:
Instead of this pattern, I think you can import a package for side effects to keep consistent with normal cobra style `import _ "image/png"`. Can you go back to the old pattern?

Reply:
This creates cyclic import between cmd and auth package which is not allowed in Go lang, unfortunately.